src/eko/__init__.py

```diff
@@ -5,3 +5,9 @@
 from .runner import solve
 
 __version__ = version.__version__
+
+__all__ = [
+    "io",
+    "EKO",
+    "solve",
+]
```

Comment:
for future reference: the linter also required me to add these lines (and similar in other places, e.g. `ekobox`) - which are a step towards #145

Reply:
Ok, PEP8 is pretty much succinct about the topic...

https://peps.python.org/pep-0008/#public-and-internal-interfaces